packages/babel-plugin-transform-es2015-classes/src/index.js

```diff
@@ -3,10 +3,15 @@ import VanillaTransformer from "./vanilla";
 import annotateAsPure from "babel-helper-annotate-as-pure";
 import nameFunction from "babel-helper-function-name";
 
-export default function({ types: t }) {
+export default function({ types: t }, options) {
+  const { loose } = options;
+
   // todo: investigate traversal requeueing
   const VISITED = Symbol();
 
+  let Constructor = VanillaTransformer;
```

Comment:
atm there are 2 code styles regarding this:
```js
// 1
const buildClassProperty = loose
  ? buildClassPropertyLoose
  : buildClassPropertySpec;
```
and
```js
let Constructor = VanillaTransformer;
if (loose) Constructor = LooseTransformer;
```

I would choose one 1 of them (my choice would be the former one) and unify it across the plugins

Reply:
@Andarist 
I'm totally new to project. Could you please clarify what exactly is the former one?

Reply:
@Andarist is just suggesting being consistent in the areas where we do something based on loose mode (often a template or function). FWIW, I too prefer the ternary.